db/migrate/20210203021559_create_location.rb

```diff
@@ -2,6 +2,10 @@ class CreateLocation < ActiveRecord::Migration[5.2]
   def change
     create_table :locations do |t|
       t.string :name
+      t.integer :population
+      t.boolean :urban
+      t.timestamp :created_at
+      t.timestamp :updated_at
```

Comment:
Great job with getting the additional attributes added for Locations! I do not want to get into to much detail tonight with this specific update to the migration and think we can review this together first thing tomorrow morning however I think to add these attributes or new columns to the table, you would want to create a new migration that adds the columns instead of changing the original migration you have.  That way you are not going back in time and  altering the history of what you originally had, but instead showing the additions building up overtime.